pkg/apis/management/v1/management.proto

```diff
@@ -135,6 +146,30 @@ service Management {
       get: "/rbac/backend/{name}/roles"
     };
   }
+  rpc AddAdminRoleBinding(core.Reference) returns (google.protobuf.Empty) {
```

Comment:
Making these separate apis entirely seems like the wrong move, since when we undo this it will necessarily be a breaking change. Is there a better way?

Reply:
I don't think it will be breaking.  Once we create more granular permissions this would simply use a different RPC